master/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml

```diff
@@ -12,9 +12,9 @@ data:
   # defined below.
   etcd_endpoints: "http://10.96.232.136:6666"
 
-  # True enables BGP networking, false tells Calico to enforce
+  # Disable Calico networking backend, which tells Calico to enforce
   # policy only, using native networking.
-  enable_bgp: "true"
+  calico_backend: "none"
```

Comment:
Why set this variable at all? It defaults to etcd. Is it because we're aiming for a similar looking yaml file for each installations and  a different config map for each?